test/micro/org/openjdk/bench/java/lang/foreign/xor/XorOp.java

```diff
@@ -4,4 +4,8 @@ public interface XorOp {
 
     void xor(byte[] src, int sOff, byte[] dst, int dOff, int len) throws Throwable;
 
+<<<<<<< HEAD
+    void copy(int count, byte[] src, int sOff, byte[] dst, int dOff, int len);
+=======
+>>>>>>> 9727f4bdddc071e6f59806087339f345405ab004
```

Comment:
You have multiple merge conflicts in the micro benchmark files.

Reply:
Sorry, not sure how I missed the conflicts.  They should be resolved now. Thanks!